src/components/pages/Offers/Offer/Stocks/ActivationCodesUploadDialog/ActivationCodesCsvForm/ActivationCodesCsvForm.jsx

```diff
@@ -2,23 +2,43 @@ import PropTypes from 'prop-types'
 import React, { forwardRef, Fragment } from 'react'
 
 export const ActivationCodeCsvForm = forwardRef(function ActivationCodeCsvForm(
-  { isFileInputDisabled, submitThumbnail },
+  { isFileInputDisabled, submitThumbnail, errorMessage },
   ref
 ) {
   return (
     <Fragment>
-      <div className="activation-codes-upload-description">
-        <p>
-          {
-            'Pour les offres nécessitant une activation par code sur une plateforme extérieure, vous pouvez importer directement un fichier .csv.'
-          }
-        </p>
-        <p>
-          {
-            'Le stock disponible sera automatiquement mis à jour. Les jeunes auront accès à ce code dans leur espace réservation.'
-          }
-        </p>
-      </div>
+      {!errorMessage && (
+        <div className="activation-codes-upload-description">
+          <p>
+            {
+              'Pour les offres nécessitant une activation par code sur une plateforme extérieure, vous pouvez importer directement un fichier .csv.'
+            }
+          </p>
+          <p>
+            {
+              'Le stock disponible sera automatiquement mis à jour. Les jeunes auront accès à ce code dans leur espace réservation.'
+            }
+          </p>
+        </div>
+      )}
+      {errorMessage && (
+        <div className="activation-codes-upload-description">
+          <p>
+            {"Une erreur s'est produite lors de l'import de votre fichier "}
```

Comment:
(typo / détail)

```suggestion
            {"Une erreur s'est produite lors de l'import de votre fichier."}
```

... pour être homogène avec les autres messages d'erreur qui finissent par un point.

Reply:
En fait le point est un peu plus bas du fait que le nom du fichier est en gras et le point suivant pas en gras.
Comme je voulais pas interpreter du html (La méthode react qui fait ça s'appelle d'ailleurs dangerouslySetInnerHTML ^^), la concaténation est un peu chelou.
```react
          <p>
            {"Une erreur s'est produite lors de l'import de votre fichier "}
            <b>
              {`${ref.current.files[0].name}`}
            </b>
            {'.'}
          </p>
```

Reply:
Oh, effectivement ! Je n'avais pas bien fait attention.